website/src/docs/aws-s3-multipart.md

```diff
@@ -109,20 +111,31 @@ A function that generates a batch of signed URLs for the specified part numbers.
  - `key` - The object key in the S3 bucket.
  - `partNumbers` - An array of indecies of this part in the file (`PartNumber` in S3 terminology). Note that part numbers are _not_ zero-based.
 
-Return a Promise for an object with keys:
+`prepareUploadParts` should return a `Promise` with an `Object` with keys:
 
  - `presignedUrls` - A JavaScript object with the part numbers as keys and the presigned URL for each part as the value. An example of what the return value should look like:
-
-   ```js
-   // for partNumbers [1, 2, 3]
-   return {
-     1: 'https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=1&...',
-     2: 'https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=2&...',
-     3: 'https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=3&...',
-   }
-   ```
  - `headers` - **(Optional)** Custom headers that should be sent to the S3 presigned URL.
 
+```json
+{
+  "presignedUrls": {
+    "1": "https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=1&...",
+    "2": "https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=2&...",
+    "3": "https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=3&..."
+  },
+  "headers": { "some-header": "value" }
+}
```

Comment:
```suggestion
{
  presignedUrls: {
    1: 'https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=1&...',
    2: 'https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=2&...',
    3: 'https://bucket.region.amazonaws.com/path/to/file.jpg?partNumber=3&...',
  },
  headers: { 'some-header': 'value' },
}
```

Reply:
I don't think the linter is going to like that. FWIW I think JSON is fine here.

Reply:
Oh I didn't know that we also lint js blocks inside markdown.

Reply:
I'm also inclined to keep it the way it is.